platformio.ini

```diff
@@ -30,34 +31,46 @@ platform = native@^1.1.3
 extra_scripts = support/sdl2_build_extra.py
 build_flags =
   ${env.build_flags}
-  -D LV_LOG_LEVEL=LV_LOG_LEVEL_INFO
+  ; -D LV_LOG_LEVEL=LV_LOG_LEVEL_INFO
   -D LV_LOG_PRINTF=1
   ; Add recursive dirs for hal headers search
   !python -c "import os; print(' '.join(['-I {}'.format(i[0].replace('\x5C','/')) for i in os.walk('hal/sdl2')]))"
   -lSDL2
   ; SDL drivers options
   -D LV_LVGL_H_INCLUDE_SIMPLE
   -D LV_DRV_NO_CONF
-  -D USE_MONITOR
-  -D MONITOR_ZOOM=2
-  -D USE_MOUSE
-  -D USE_MOUSEWHEEL
-  -D USE_KEYBOARD
+  -D USE_SDL
+  -D SDL_HOR_RES=480
+  -D SDL_VER_RES=320  
+  -D SDL_ZOOM=1
+  -D SDL_INCLUDE_PATH="\"SDL2/SDL.h\""
+  -D LV_MEM_CUSTOM=1
+  -D LV_MEM_SIZE="(128U * 1024U)"
+  ; -D USE_MONITOR
+  ; -D MONITOR_ZOOM=2
+  ; -D USE_MOUSE
+  ; -D USE_MOUSEWHEEL
+  ; -D USE_KEYBOARD
 lib_deps =
   ${env.lib_deps}
-  lv_drivers@~7.9.0
-  ;lv_drivers=https://github.com/littlevgl/lv_drivers/archive/7d71907c1d6b02797d066f50984b866e080ebeed.zip
+  ; Use direct URL, because package registry is unstable
+  ;lv_drivers@~7.9.0
+  lv_drivers=https://github.com/lvgl/lv_drivers/archive/refs/tags/v8.2.0.zip
 src_filter =
   +<*>
   +<../hal/sdl2>
+  +<../.pio/libdeps/emulator_64bits/lvgl/demos>
```

Comment:
Probably worth add comment before such lines `; this is specific for demo, remove when copy file to your project`

Reply:
No need, this is the path to include the demos folder download automatically from the github, and the path will not change unless platformio changes it (it happens in the past).  Removing it will cause linking error since the demos folder will not be included for compilation automatically.  However, if you have a better way to force compile the demo s folder, please let me know.  currently I can only make it works this way.

Reply:
You missed my point. This line should exist now. But main goal of project is to provide easy-to-use samples of files. So, if any line in config is not obvious for beginer - comment can help. I do not not insist, only propose.

Reply:
Added comments for demo lines.